cal/cal/static/css/calendaranalytics.css

```diff
@@ -15,6 +15,20 @@ h1, h2, h3, h4, h5, h6, a, p {
 button {
   margin: 2px; }
 
+.section-info {
+  font-size: 14px;
+  background-color: #fff;
+  border: 1px solid #000;
+  border-radius: 5px;
+  padding: 10px;
+  margin: auto;
+  line-height: normal;
+}
+
+.fc-content {
```

Comment:
Ahh, makes sense. May I suggest you write in a comment so anyone in the future knows why that's there?